src/js/components/Header.js

```diff
@@ -65,20 +79,26 @@ class Header extends Component {
           />
           <ul>
             <li>
-              <a href="/">
-                Get Advice{' '}
-              </a>
+              <a href="/">Get Advice </a>
```

Comment:
Is the space after `Get Advice` intentional? Otherwise, we should just remove it. 👾